### ISSUE TITLE
Lycopodiums use Dream Flower when they shouldn't

### ISSUE THREAD

**Issue by Era-Lusiphur** [project-topaz/topaz#983]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Lycopodiums are currently coded to use the typical Mandragora skill list. This includes Dream Flower, which they apparently should never use. I'm not certain what their full skill list should be, so I'm not submitting a PR as of this time. If Dream Flower is removed, it makes the Rolanberry Fields (S) Lycopodiums an attractive (S) camp prior to the East Ronfaure (S) colibri camp.

Reference: http://campsitarus.blogspot.com/2008/05/23-25-rolanberry-fields-s.html

>  Another big draw to Lycopodiums is that Lycopodiums don't use Dream Flower.

**Comment by Era-Lusiphur** [comment]
I just created a new list on Era removing Dream Flower. If others are content with that, more than happy to submit a quick PR for it.

**Comment by TeoTwawki** [comment]
yeah they get that petal something move that takes away tp instead of dream flower

**Comment by Era-Lusiphur** [comment]
Do you know if all of the other TP moves are the same?

**Comment by Era-Lusiphur** [comment]
@TeoTwawki Are you saying they use Petal Pirouette? FFXIclopedia definitely began it's decline post-WotG due to politics, so it's not 100% clear from just wikis what their skill list might be. I don't see any videos or captures for it either.

https://ffxiclopedia.fandom.com/wiki/Petal_Pirouette
https://ffxiclopedia.fandom.com/wiki/Category:Mandragora
https://www.bg-wiki.com/bg/Lycopodium_(Monster)
https://ffxi.gamerescape.com/wiki/Lycopodium
https://www.zentoo.net/ffxi/blu/DreamFlower.html

**Comment by TeoTwawki** [comment]
In the case of some NM Lycos its the only move they get at all. I was pretty sure the normies get it in place of dream flower...We're going to need testers to go feed one tp because ffxiclopedia is claiming only the NMs have it (and bg doesn't specify at all).

**Comment by TeoTwawki** [comment]
In any case the fix for dream flower is: make a new list that omits dream flower. give that to lyco pools. done.
